package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "stacked-sheet",
+  "main": "src/stacked-sheet.js",
+  "module": "src/stacked-sheet.js",
+  "files": [
+    "src"
+  ],
+  "version": "0.0.0",
+  "description": "Webcomponent stacked-sheet with lit-element and open-wc recommendations",
+  "license": "MIT",
+  "dependencies": {
+    "lit-element": "^2.0.1"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "scripts": {
+    "lint": "npm run lint:eslint && npm run lint:prettier",
```

Comment:
I think we need to change how we use lint here. Because in the pipeline, we only run `lint` script and if we don't run `format` script before, build will fail. So it would be better if we have only one script for lint.

Reply:
I am not sure what consensus is here but I think a developer should run lint before checking in the code (or, alternatively, pay attention to problems highlighted in the IDE) and fix the errors (either by using format or not).

Reply:
Fixed CRLF\LF issue. Builds should work now 👍

Reply:
👍